### DISCUSSION TITLE
What's the difference between Avalonia.Svg.Svg and Avalonia.Svg.Skia.Svg?

### DISCUSSION THREAD

**Discussion by ds1709** [wieslawsoltes/Svg.Skia#230] [Q&A]
What's the difference between Avalonia.Svg.Svg and Avalonia.Svg.Skia.Svg? Why both are exists? When I should use the first or the second one?
Also! I noticed, that some svg images doesn't render with Avalonia.Svg.Svg, but render correctly with Avalonia.Svg.Skia.Svg. Is it bug or by-design?

**Comment by wieslawsoltes** [comment]
Avalonia.Svg renders using Avalonia DrawingContext primitives and works on all supported Avalonia platforms, Avalonia.Svg.Skia uses SkiaSharp and requires SkiaSharp support enabled in Avalonia (which is by default enabled).
